cdap-distributions/src/hdinsight/pkg/install.sh

```diff
@@ -22,9 +22,15 @@
 # The git branch to clone
 CDAP_BRANCH='release/4.1'
 # Optional tag to checkout - All released versions of this script should set this
+<<<<<<< HEAD
 CDAP_TAG='v4.1.0'
 # The CDAP package version passed to Chef
 CDAP_VERSION='4.2.0-1'
+=======
```

Comment:
basically we should just set it to what will be appropriate after release... ie HEAD version but with tag updated to v4.2.0